### PR TITLE
ICU-20473 Adding collation ucadata option to filter file.

### PR DESCRIPTION
**Pull request by sffc**
I added an option to the config file.  Do you prefer also removing the command-line flag?

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-20473
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

**Comment by markusicu** [comment]
> Do you prefer also removing the command-line flag?

Yes, I think so.
1. I think it's cleaner to do this via the config file.
2. I don't like two ways of doing the same thing. (If we can help it.)

File: icu4c/source/data/buildtool/filtration_schema.json [hunk 1]
```diff
@@ -53,6 +53,10 @@
             },
             "additionalProperties": false,
             "required": ["directory", "replacements"]
+        },
+        "collationUCAData": {
```
Comment (markusicu) [inline icu4c/source/data/buildtool/filtration_schema.json]: userguide/icu_data/buildtool.md shows a category "coll_ucadata". Can/should this option be in/on that category?

Comment (sffc) [reply icu4c/source/data/buildtool/filtration_schema.json]: Discussed.

**Comment by sffc** [comment]
Fixed via amend. PTAL

**Comment by sffc** [comment]
Note: the docs need to be in a different PR currently because the docs are on a different branch (master vs maint/maint-64).  Normally, the docs would be included in the same PR.

**Pull request merged**